pytorch/training/docker/1.13/py3/Dockerfile.sagemaker.lite.cpu

```diff
@@ -148,50 +142,47 @@ RUN pip install --upgrade pip --trusted-host pypi.org --trusted-host files.pytho
     "pyyaml>=5.4,<5.5" \
     scipy
 
-COPY deep_learning_container.py /usr/local/bin/deep_learning_container.py
+# TODO: Install deep_learning_container.py here
 
-RUN chmod +x /usr/local/bin/deep_learning_container.py
+# TODO: Install PyTorch license here
 
-RUN curl -o /license.txt  https://aws-dlc-licenses.s3.amazonaws.com/pytorch-1.12/license.txt
+########################################################
+#  _____ ____ ____    ___
+# | ____/ ___|___ \  |_ _|_ __ ___   __ _  __ _  ___
+# |  _|| |     __) |  | || '_ ` _ \ / _` |/ _` |/ _ \
+# | |__| |___ / __/   | || | | | | | (_| | (_| |  __/
+# |_____\____|_____| |___|_| |_| |_|\__,_|\__, |\___|
+#                                         |___/
+#  ____           _
+# |  _ \ ___  ___(_)_ __   ___ 
+# | |_) / _ \/ __| | '_ \ / _ \
+# |  _ <  __/ (__| | |_) |  __/
+# |_| \_\___|\___|_| .__/ \___|
+#                  |_|
+########################################################
 
-##############################################################################
-#  _____ _____   ___                              ____           _
-# | ____|___ /  |_ _|_ __ ___   __ _  __ _  ___  |  _ \ ___  ___(_)_ __   ___
-# |  _|   |_ \   | || '_ ` _ \ / _` |/ _` |/ _ \ | |_) / _ \/ __| | '_ \ / _ \
-# | |___ ___) |  | || | | | | | (_| | (_| |  __/ |  _ <  __/ (__| | |_) |  __/
-# |_____|____/  |___|_| |_| |_|\__,_|\__, |\___| |_| \_\___|\___|_| .__/ \___|
-#                                    |___/                        |_|
-##############################################################################
-
-FROM common AS e3
+FROM common AS ec2
 
 ARG PYTHON
 
-# PyTorch Binaries
-ARG PT_E3_TRAINING_URL
-ARG PT_TORCHVISION_URL
-ARG PT_TORCHAUDIO_URL
-ARG PT_TORCHDATA_URL
-ARG PT_S3_WHL
+# TODO: Redeclare ARGs for PyTorch and other Torch packages here
 
-# Install AWS-PyTorch and other torch packages
-RUN pip uninstall -y torch torchvision torchaudio torchdata \
- && pip install --no-cache-dir -U ${PT_E3_TRAINING_URL} ${PT_TORCHVISION_URL} ${PT_TORCHAUDIO_URL} ${PT_TORCHDATA_URL}
+# uninstall if already present
+RUN pip uninstall -y torch torchvision torchaudio torchdata
 
-# Install PT S3 plugin
-RUN pip install -U ${PT_S3_WHL}
-RUN mkdir -p /etc/pki/tls/certs && cp /etc/ssl/certs/ca-certificates.crt /etc/pki/tls/certs/ca-bundle.crt
+# TODO: Install AWS-PyTorch and other torch packages here
+
+RUN /opt/conda/bin/conda install -c conda-forge \
```

Comment:
This should be `-c pytorch-nightly` -- https://anaconda.org/pytorch-nightly/repo

Reply:
Changed.